common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -433,39 +444,11 @@ def parse_query_params(strategy, response, *args, **kwargs):
     if not (auth_entry and auth_entry in _AUTH_ENTRY_CHOICES):
         raise AuthEntryError(strategy.backend, 'auth_entry missing or invalid')
 
-    # Note: We expect only one member of this dictionary to be `True` at any
-    # given time. If something changes this convention in the future, please look
-    # at the `login_analytics` function in this file as well to ensure logging
-    # is still done properly
```

Comment:
Thank you @wedaly.  I have now manually tested the reg/login flow on my local machine.  And we will retest on stage before release.